Editor/Scripts/CodeGenerators/GenerateRGActionMapClass.cs

```diff
@@ -0,0 +1,119 @@
+using UnityEngine;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using UnityEditor;
+
+namespace RegressionGames
+{
+    public static class GenerateRGActionMapClass
```

Comment:
A comment showing what exactly this generates would be helpful!

Reply:
Added 👍